### PR TITLE
SPCTR-2916 Presets Global UX Changes and Settings Rearrangement

### PR DESCRIPTION
**Pull request by aapatre**
### Description
- New Preset Icons
- Presets panels are shifted as per priorities
- Some settings controls are rearranged too as per priorities and special cases
- [JIRA](https://brainstormforce.atlassian.net/browse/SPCTR-2916)

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/tabs/settings.js [hunk 1]
```diff
@@ -245,8 +246,9 @@ const Settings = ( props ) => {
 				},
 			],
 		};
+
 		return (
-			<UAGAdvancedPanelBody title={ __( 'Layout', 'ultimate-addons-for-gutenberg' ) } initialOpen={ false }>
+			<UAGAdvancedPanelBody title={ __( 'Layout & Tabs Title', 'ultimate-addons-for-gutenberg' ) } initialOpen={ true }>
```
Comment (bsf-zanev) [inline src/blocks/tabs/settings.js:251]: This may be too big a title.

Would `Layout` encompass it all?

**Review by bsf-zanev — COMMENTED** [review]
A few minor comments.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Spectra - WordPress Gutenberg Blocks #
-**Contributors:** [brainstormforce](https://profiles.wordpress.org/brainstormforce/)  
+**Contributors:** [brainstormforce](https://profiles.wordpress.org/brainstormforce)
```
[the remark this answers is not in the document]
Comment (aapatre) [reply README.md:2]: This is auto-generated, not done manually

File: README.md [hunk 3]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Improvement: Global - Presets and Settings UX - Fresh new look for presets icons and also rearranged settings.
```
[the remark this answers is not in the document]
Comment (aapatre) [reply README.md]: Done

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (aapatre) [reply README.md:2]: I tried adding it back and ran `grunt readme`, it's removing the slash

File: src/blocks/tabs/settings.js (hunk 1 above)
Comment (aapatre) [reply src/blocks/tabs/settings.js]: Changed to 'Layout'

File: src/blocks/blockquote/settings.js [hunk 4]
```diff
@@ -1390,7 +1220,7 @@ const Settings = ( props ) => {
 								setAttributes={ setAttributes }
 							/>
 						}
-						disableBottomSeparator={ true }
+						disableBottomSeparator={ false }
```
[the remark this answers is not in the document]
Comment (aapatre) [reply src/blocks/blockquote/settings.js]: Done

**Review by bsf-zanev — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

**Pull request merged**